scrapy_poet/utils.py

```diff
@@ -1,24 +1,10 @@
 import os
+from pathlib import PosixPath
 
 from scrapy.utils.project import project_data_dir, inside_project
-from tldextract import tldextract
 
 
-def get_domain(url):
-    """
-    Return the domain without any subdomain
-
-    >>> get_domain("http://blog.example.com")
-    'example.com'
-    >>> get_domain("http://www.example.com")
-    'example.com'
-    >>> get_domain("http://deeper.blog.example.co.uk")
-    'example.co.uk'
-    """
-    return ".".join(tldextract.extract(url)[-2:])
-
-
-def get_scrapy_data_path(createdir: bool = True, default_dir: str = ".scrapy") -> str:
+def get_scrapy_data_path(createdir: bool = True, default_dir: str = ".scrapy") -> PosixPath:
```

Comment:
Is it PosixPath on Windows as well?

Reply:
Nice catch! Fixed this in https://github.com/scrapinghub/scrapy-poet/pull/56/commits/05881057934c126a26a1232f981fd9928ac85f91.